README.md

```diff
@@ -6,30 +6,28 @@ An _experimental_ (Read: not-usable or in anyway done) distributed podcasting pl
 
 Getting started:
 
-Install the vendor dependencies using godep. (https://github.com/golang/dep)
+We use [fresh](https://github.com/pilu/fresh) for hot code reloading with docker-compose.
```

Comment:
Doesn't fresh only work with traffic, martini or gocraft/web? We may be able to use that, though at the moment we're not.

Reply:
Fresh works with any changes.

Reply:
^ Dope.